packages/apple-pay-integration/types/ApplePaySession.d.ts

```diff
@@ -0,0 +1,7 @@
+export {};
+
+declare global {
+  interface Window {
+    ApplePaySession: ApplePaySession
```

Comment:
I believe ApplePay isn't defined for every browser so it is more accurate to put a question mark in front.
```suggestion
    ApplePaySession?: ApplePaySession
```

Reply:
good point, will update